src/ocm/services/ClustersService.ts

```diff
@@ -59,6 +60,11 @@ const ClustersService = {
     }
     return params;
   },
+
+  async pollCluster(clusterId: Cluster['id']) {
```

Comment:
```suggestion
  async get(clusterId: Cluster['id']) {
```
This is not really "polling", I would just call it "get", otherwise whoever will use this function in the future may think it will be do some sort of timed polling, when in reality all it does is just a get request.